repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-untranslated-folder.yaml

```diff
@@ -0,0 +1,24 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:queries/hippo:templates/new-untranslated-folder:
```

Comment:
Where is this used?

Reply:
https://www.onehippo.org/library/concepts/document-types/template-queries.html